app/src/ingestion/edd/cache.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+"""
+Adapted from https://github.com/scrapy-plugins/scrapy-splash/tree/master/scrapy_splash
+
+To handle "splash" Request meta key correctly when HTTP cache is enabled
+Scrapy needs a custom caching backed.
+
+See https://github.com/scrapy/scrapy/issues/900 for more info.
+"""
+from __future__ import absolute_import
+
+import os
+from datetime import datetime
+
+from scrapy.extensions.httpcache import FilesystemCacheStorage
+from scrapy.http.request import Request
+from scrapy.spiders import Spider
+
+# Create file
+crawl_log = open(
+    f"crawl_log-{datetime.now().strftime('%Y-%m-%d-%H%M%S')}.txt", "w", encoding="utf-8"
+)
```

Comment:
Is this file used elsewhere or for our convenience?

Reply:
Just for our development/learning purposes.